### ISSUE TITLE
[Investigate] Better transition support

### ISSUE THREAD

**Issue by lwis** [ggravlingen/pytradfri#39]
Do commands other than set_dimmer support transitions?

**Comment by ggravlingen** [comment]
I've been considering if you can use the smart tasks to build custom and one-off transitions for this purpose. In smart tasks, you can set a transition time, start time and target dimmer level. Or would it be to cumbersome to use that method?

**Comment by lwis** [comment]
I haven't really used the smart tasks, I'm not sure they're a good fit for one off executions like the use cases HASS has.

**Comment by ggravlingen** [comment]
No, you're probably right. You would essentially need to create a smart task which would be immediately executed and then delete the task. Otherwise, it'll show up in the app, which might not be expected behaviour.

**Comment by lwis** [comment]
Are there any schemas for the protocol? Wondering if there's any way of 'exploring' what the Gateway exposes.

**Comment by ggravlingen** [comment]
You basically get the JSON-string back that we have here in TASK: https://github.com/ggravlingen/pytradfri/blob/master/tests/test_smart_task.py

**Comment by ggravlingen** [comment]
You can of course also poll the whole smart task-path under /15010 (CONST=ROOT_SMART_TASKS).

**Comment by myusuf3** [comment]
any progress on this? I would love to be able to fade in lights after a movie or fade out.

**Comment by lwis** [comment]
@myusuf3 both work already by using `set_dimmer`. My branch in HA supports it.

**Comment by myusuf3** [comment]
@lwis has this been merged into HA? create a PR I will see it gets merged in. /cc @robbiet480

**Comment by lwis** [comment]
home-assistant/home-assistant#7815 

It's an old one, feel free to get involved to get it unblocked.

**Comment by ggravlingen** [comment]
@lwis guess we can close this issue now as all of your code references has been merged?
